common/wonder_upgrades/wc_wonders_historical_upgrades.txt

```diff
@@ -1,5 +1,148 @@
 #Last Modified: 7/11/2019
 
+upgrade_old_god_cthun_1 = {
+	picture = GFX_upgrade_icon_old_god_cthun
+
+	potential = { has_wonder_upgrade =  upgrade_old_god_cthun_1 }
+	allow = { }
+	active = { }
+
+	flags = {
+		is_singular
+	}
+	build_time = 1
+	gold_cost_flat = 1
+	gold_cost_ticking = 1
+	local_modifier = {
+		garrison_size = 5
+		heavy_infantry = 150
+		fort_level = 5
+	}
+	owner_modifier = {
+		monthly_character_prestige = 1
+		monthly_character_piety = 1
+		monthly_character_wealth = 8
+	}
+}
+upgrade_old_god_cthun_2 = {
+	picture = GFX_upgrade_icon_old_god_cthun
+
+	potential = { has_wonder_upgrade =  upgrade_old_god_cthun_2 }
+	allow = { }
+	active = { }
+
+	flags = {
+		is_singular
+	}
+	build_time = 24
+	gold_cost_flat = 400
+	gold_cost_ticking = 1
+	local_modifier = {
+		garrison_size = 5
+		heavy_infantry = 300
+		fort_level = 10
+	}
+	owner_modifier = {
+		monthly_character_prestige = 2
```

Comment:
We arent doing prestigous thing, instead of prestige we can got **Shat'Gral Opinion 10.**